classes/blocks/class-covers.php

```diff
@@ -144,6 +144,9 @@ public function __construct() {
 						'type'    => 'string',
 						'default' => self::GRID_LAYOUT,
 					],
+					'readMoreText'     => [
+						'type' => 'string',
+					],
```

Comment:
I tested it locally (can replicate the issue), you also need to add a few lines for existings blocks, in the `render_callback` around l.94
```php
		if ( empty( $attributes['readMoreText'] ) ) {
			$attributes['readMoreText'] = __( 'Load more', 'planet4-blocks' );
		}

```